src/components/PostList/PostList.tsx

```diff
@@ -1,6 +1,16 @@
 import React from 'react';
+import { PrepearedPost } from '../../types/PrepearedPosts';
+import { PostInfo } from '../PostInfo/PostInfo';
+import { UserInfo } from '../UserInfo/UserInfo';
 import './PostList.scss';
 
-export const PostList: React.FC = () => (
-  <>Put the list here</>
+interface Props {
+  post: PrepearedPost,
+}
+
+export const PostList: React.FC<Props> = ({ post }) => (
+  <>
+    <PostInfo post={post} />
+    {(post.user) && <UserInfo user={post.user} /> }
```

Comment:
PostList component should accept an array of posts and render PostInfo component for each post
UserInfo component should be rendered inside the PostInfo component, just delete it from here and use map for rendering PostInfo components